abs.yaml

```diff
@@ -0,0 +1,2 @@
+upload_channels:
+  - sfe1ed40
```

Comment:
https://anaconda.atlassian.net/browse/PKG-8064
requests that it be uploaded to defaults. Can we change this